test/cql-pytest/cassandra_tests/validation/operations/select_test.py

```diff
@@ -1298,7 +1298,7 @@ def testIndexQueryWithValueOver64K(cql, test_keyspace):
         assert_empty(execute(cql, table, "SELECT * FROM %s WHERE c = ?  ALLOW FILTERING", too_big))
 
 # Reproduces #10366
-@pytest.mark.xfail(reason="#10366 - server error instead of InvalidRequest")
+pytest.mark.xfail(reason="#10366 - server error instead of InvalidRequest")
```

Comment:
I think you have a typo here - you should have deleted this entire xfail line, and instead just deleted the "@". I wonder how this even works?